packages/p2p-v2/src/components/Clipboard/Clipboard.tsx

```diff
@@ -0,0 +1,40 @@
+//TODO: to be replaced with derivcom component
+import React, { useEffect, useState } from 'react';
+import { useCopyToClipboard } from 'usehooks-ts';
+import { Tooltip } from '@deriv-com/ui';
+import CheckmarkCircle from '../../public/ic-checkmark-circle.svg';
+import CopyIcon from '../../public/ic-clipboard.svg';
+import './Clipboard.scss';
+
+type TClipboardProps = {
+    textCopy: string;
+};
+
+const Clipboard = ({ textCopy }: TClipboardProps) => {
+    const [, copy] = useCopyToClipboard();
+    const [isCopied, setIsCopied] = useState(false);
+    let timeoutClipboard: ReturnType<typeof setTimeout>;
+
+    const onClick = (event: { stopPropagation: () => void }) => {
```

Comment:
@nada-deriv, does the type `React.MouseEvent` work here?
```suggestion
    const onClick = (event: MouseEvent) => {
```